modAion/src/org/aion/zero/types/AionTransaction.java

```diff
@@ -246,21 +258,23 @@ public void setTimeStamp(long timeStamp) {
     }
 
     @Override
-    public Address getTo() {
+    public Address getDestinationAddress() {
         if (!parsed) {
             rlpParse();
         }
         return to;
     }
 
+    @Override
     public byte[] getData() {
         if (!parsed) {
             rlpParse();
         }
         return data;
     }
 
-    public byte getType() {
+    @Override
+    public byte getTargetVM() {
```

Comment:
just think we might have different transaction type implement. Is it okay to use it for the target VM?

Reply:
Yes, I was also thinking that `getTargetVM` may be too restrictive also. Currently transactions have a notion of a `type` and a `kind`, and this is really confusing and non-informative, so I would like to improve one of them. I'll try and think through some ideas, maybe `kind` should be the one to change